Dockerfile

```diff
@@ -3,6 +3,8 @@ FROM python:3.8.12-buster as python-base
 LABEL maintainer="TACC-ACI-WMA <wma_prtl@tacc.utexas.edu>"
```

Comment:
Would recommend changing line 1 of this file to:
`FROM python:3.8-buster as python-base`

in order to always pick up patch versions of 3.8 on new builds

Reply:
Okay. That's safe. Will do.

<details>

I had to double check we can safely go higher than 3.8.12. We can, but if we want to go to 3.9, we need to update our Django to 2.2.17 (not scary, because we are at 2.2.16). ([Python Compatibility](https://docs.djangoproject.com/en/3.2/releases/2.2/#python-compatibility))

_P.S. Django is updating packages for its v4 release, so we should consider upgrading to Django v3._

</details>

Reply:
https://github.com/TACC/Core-CMS/pull/472